packages/lifecycle/spec/LifeCycleBehaviors.ts

```diff
@@ -48,11 +50,12 @@ describe('KO LifeCycle', function () {
       assert.isFunction(Child.prototype.anchorTo)
       assert.isFunction(Child.prototype.dispose)
       assert.isFunction(Child.prototype.addDisposable)
-      assert.isNotFunction(Child.prototype.mixInto)
+      // TODO: Fails with tsc, check if test is obsolte with typescript
+      // assert.isNotFunction(Child.prototype.mixInto)
```

Comment:
Noted.  Since it's unlikely to be used and a strange/rare pattern, this feels ok to me.

Reply:
Alternatively we could use any or set ts-ignore for the `isNotFunction` check?